_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -33150,18 +33150,14 @@
 /turf/open/floor/plasteel,
 /area/security/main)
 "bnS" = (
-/obj/structure/cable,
-/obj/effect/turf_decal/tile/neutral{
-	dir = 1
-	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer4{
-	dir = 4
+/obj/machinery/camera{
+	c_tag = "Holodeck - Fore 2";
+	name = "holodeck camera"
 	},
-/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden/layer2{
-	dir = 1
+/turf/open/floor/engine{
+	name = "Holodeck Projector Floor"
 	},
```

Comment:
You removed a cable, pipe and scrubber from this turf, but they don't APPEAR to be duplicates. Can you double check this for me? My DMM-reading-fu isn't always the best.

Reply:
I've taken a look over and can't find this. I suspect this is just one of those weird reorganisations you get with .TGM file occasionally.